lib/code_corps/stripe/adapters/stripe_card.ex

```diff
@@ -0,0 +1,9 @@
+defmodule CodeCorps.Stripe.Adapters.StripeCard do
+  import CodeCorps.MapUtils, only: [rename: 3]
+
+  def params_from_stripe(%{} = stripe_map) do
+    stripe_map
+    |> rename("id", "id_from_stripe")
+    |> rename("customer", "stripe_customer_id")
```

Comment:
Also, it looks like `customer` could really vary here. Sometimes it may be `null`. We probably don't want to be attempting anything really crazy.